src/UKHO.S100PermitService.API/appsettings.local.overrides.json

```diff
@@ -0,0 +1,65 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    },
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Information"
+      }
+    }
+  },
+  "EventHubLoggingConfiguration": {
+    "MinimumLoggingLevel": "Warning",
+    "UkhoMinimumLoggingLevel": "Information",
+    "Environment": "dev",
+    "EntityPath": "",
+    "System": "S100 API",
+    "Service": "S100 Permit Service",
+    "NodeName": "Azure",
+    "ConnectionString": ""
+  },
+  "KeyVaultSettings": {
+    "ServiceUri": ""
+  },
+  "ApplicationInsights": {
+    "ConnectionString": ""
+  },
+  "ManufacturerKeyVault": {
+    "ServiceUri": "",
+    "CacheTimeoutInHours": 2
+  },
+  "Swagger": {
+    "Version": "0.0.1",
+    "Title": "S100 Permit Service APIs",
+    "Description": "This API is for the S100 Permit Service.",
+    "Email": "",
+    "Route": "swagger"
+  },
+  "AzureAdConfiguration": {
+    "MicrosoftOnlineLoginUrl": "https://login.microsoftonline.com/",
+    "TenantId": "",
+    "ClientId": ""
+  },
+  "AllowedHosts": "*",
+  "PermitServiceManagedIdentityConfiguration": {
+    "DeductTokenExpiryMinutes": 1
+  },
+  "HoldingsServiceApiConfiguration": {
+    "ClientId": "",
+    "BaseUrl": "",
+    "RequestTimeoutInMinutes": 5
+  },
+  "UserPermitServiceApiConfiguration": {
+    "ClientId": "",
+    "BaseUrl": "",
+    "RequestTimeoutInMinutes": 5
+  },
+  "ProductKeyServiceApiConfiguration": {
+    "BaseUrl": "",
+    "ClientId": "",
+    "RequestTimeoutInMinutes": 5
+  }
+}
```

Comment:
Just considering the ins and outs of committing this file to the repo? Its intended to be a local file and although I cant see any sensitive information in this file wonder whether somebody could add some at a later date and then commit by accident?

I am aware that these files have been committed in other projects in the past!

Note:
It is generally not advisable to commit the appsettings.local.overrides.json file into a repository. Here are the reasons why:

Reasons to Avoid Committing:
1.	Sensitive Information: This file often contains sensitive information such as API keys, connection strings, and secrets that should not be exposed in a public or shared repository.
2.	Local Customizations: The settings in this file are typically specific to a developer's local environment and may not be relevant or appropriate for other environments or team members.
3.	Security Risks: Exposing sensitive configuration details can lead to security vulnerabilities.
4.

I realise this can be useful to other developers

Reply:
Agree.

Since it was pushed in some projects we thought to add it and later in upcoming PBI we were suppose to add it in .gitignore to avoid any sensitive information push just to have same settings as appsettings we pushed it. Considering the mentioned points removing in this PR.

Thanks